public/scripts/ssooverride.js

```diff
@@ -83,33 +89,53 @@
         }, false);
       });
 
-      var thimbleAuth = new WebmakerLogin({
-        csrfToken: csrf,
-        showCTA: false
-      });
+      var basicQuery = "";
+
+      basicQuery += "?client_id=" + oauthClientId;
```

Comment:
ideally, though, you'd do something like this:
```
var queryObject = {
  "client_id": oauthClientId,
  "response_type": responseType,
  "scopes": scopes
};
...
basicQuery = "?" + Object.keys(queryObj).map(function(arg) { return arg + "=" + queryObj[arg]; }).join("&")
```
So you only ever work with the structured object, until you need it serialized.